cmd/eval.go

```diff
@@ -64,7 +64,7 @@ func newEvalCommandParams() evalCommandParams {
 			evalPrettyOutput,
 			evalSourceOutput,
 		}),
-		explain: newExplainFlag([]string{explainModeOff, explainModeFull, explainModeNotes, explainModeFails}),
+		explain: newExplainFlag([]string{explainModeOff, explainModeFull, explainModeNotes, explainModeFails, explainModeDebug}),
```

Comment:
One change that came up in offline discussion was to _not_ add a new explain mode, and instead just add a new flag to disable indexing.